spec/acceptance/nodesets/default.yml

```diff
@@ -23,12 +23,12 @@ HOSTS:
     platform: el-7-x86_64
     box: centos/7
     hypervisor: vagrant
-  centos66-cli:
-    roles:
-      - client
-    platform: el-6-x86_64
-    box: puppetlabs/centos-6.6-64-nocm
-    hypervisor: vagrant
+#  centos66-cli:
```

Comment:
Uncomment and change to centos/6. It should be a 6.8 release, which supports sssd.